### DISCUSSION TITLE
Cors error when try wallet connect.

### DISCUSSION THREAD

**Discussion by Kilros0817** [WalletConnect/walletconnect-monorepo#3867] [Developer Support]
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/96753457/c9272f49-fe1f-477f-9532-1c434f2d1904)
I see this error when try to connect wallet.
But I already verified the project id.
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/96753457/bd365dd3-a3fa-470d-a0b8-e59fa3e9de02)
What is the reason?

**Comment by glitch-txs** [comment]
hi, are you still experiencing this issue?
